schemas/EGA.ArrayAssay.json

```diff
@@ -46,7 +60,7 @@
       "sample_number": {
         "type": "integer",
         "title": "Number of samples of the ArrayAssay",
-        "description": "Number of samples included in the Assay (i.e. pooled into one single microarray, labelled differently). One sample corresponds to one biological replicate [EFO:0002091] (it could be the genetic content from a single cell, a tissue… from a single individual or from several individuals). Shall not be mistaken for technical replicates [CHEBI:24432] being used several times (see https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=EGA&title=Sample+Representation).",
+        "description": "Number of samples included in the Assay (i.e. pooled into one single microarray, labelled differently). One sample corresponds to one biological replicate [EFO:0002091] (it could be the genetic content from a single cell, a tissue… from a single individual or from several individuals). Shall not be mistaken for technical replicates [EFO:0002090] being used several times (see https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=EGA&title=Sample+Representation).",
```

Comment:
```suggestion
        "description": "Number of samples included in the Assay (i.e. pooled into one single microarray, labelled differently). One sample corresponds to one biological replicate [EFO:0002091] which could be the genetic content from a single cell, a tissue from a single individual or from several individuals. Shall not be mistaken for technical replicates [EFO:0002090] being used several times (see https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=EGA&title=Sample+Representation).",
```

Reply:
A stylistic note: sometimes it's harder to read a block of text where there are multiple clauses separate out by parentheses or other types of breaks. I suggest using this formatting less and instead keep most of the text in the same "train of thought".

Reply:
I think I intended to write ``(it could be the genetic content from a single cell, a tissue…) from a single individual or from several individuals`` but placed the closing bracket where it wasn't supposed to be. 

I'll keep in mind the stylistic note.